BraveWallet/Extensions/BraveWalletSwiftUIExtensions.swift

```diff
@@ -256,4 +256,36 @@ extension BraveWallet.CoinType: Identifiable {
       return ""
     }
   }
+  
+  var defaultAccountName: String {
+    switch self {
+    case .eth:
+      return Strings.Wallet.defaultEthAccountName
+    case .sol:
+      return Strings.Wallet.defaultSolAccountName
+    case .fil:
+      return Strings.Wallet.defaultFilAccountName
+    @unknown default:
+      return ""
+    }
+  }
+  
+  var defaultSecondaryAccountName: String {
+    switch self {
+    case .eth:
+      return Strings.Wallet.defaultSecondaryEthAccountName
+    case .sol:
+      return Strings.Wallet.defaultSecondarySolAccountName
+    case .fil:
+      return Strings.Wallet.defaultSecondaryFilAccountName
+    @unknown default:
+      return ""
+    }
+  }
+}
+
+extension BraveWallet.KeyringInfo {
+  var coin: BraveWallet.CoinType? {
+    accountInfos.first?.coin
+  }
 }
```

Comment:
Not a SwiftUI specific extension (also minor nit: all the localized strings above could technically move out as well)

Reply:
fixed in [6063145](https://github.com/brave/brave-ios/pull/5705/commits/606314506064f41442b117d3566c8e2692569616)